packages/di/src/common/domain/Provider.ts

```
@@ -121,7 +122,7 @@
   }
 
   get configuration(): Partial<TsED.Configuration> {
-    return this.get("configuration");
+    return this.get("configuration")!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)